frontend/components/product-list/ProductListDeviceNavigation.tsx

```diff
@@ -1,15 +1,16 @@
-import { Flex } from '@chakra-ui/react';
+import { Flex, FlexProps } from '@chakra-ui/react';
 import { SecondaryNavbarItem, SecondaryNavbarLink } from '@components/common';
 import { IFIXIT_ORIGIN } from '@config/env';
 import { ProductList } from '@models/product-list';
 import NextLink from 'next/link';
 
-export interface ProductListDeviceNavigationProps {
+type ProductListDeviceNavigationProps = FlexProps & {
```

Comment:
I assume we weren't importing this anywhere else. We can add the `export` back later if we ever need it 👍

Reply:
Yep, wasn't being used anywhere else, so I dropped the `export` to make it clear that changing this type won't affect any other code 👍🏻